frameworks/moveos-stdlib/src/natives/moveos_stdlib/account.rs

```diff
@@ -39,8 +40,13 @@ pub fn native_create_signer(
     ty_args: Vec<Type>,
     mut arguments: VecDeque<Value>,
 ) -> PartialVMResult<NativeResult> {
-    debug_assert!(ty_args.is_empty());
-    debug_assert!(arguments.len() == 1);
+    if !ty_args.is_empty() {
+        return args_count_error(gas_params.base);
+    }
+
+    if arguments.len() != 1 {
+        return args_count_error(gas_params.base);
+    }
```

Comment:
Does the `debug_assert!` need to be changed? I think if the Move native function definition is corrected, the verifier should ensure the caller can not pass the wrong args.

Reply:
I attempted to bypass argument and type argument validation. While I could bypass it during compilation, but the verifier returns errors during module deployment.
I will remove the unnecessary validation code.